build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -74,6 +74,11 @@ export default class GenerateTypeTestsCommand extends BaseCommand<
                 "Resets the broken type test settings in package.json. Only applies to the prepare phase.",
             exclusive: ["generate"],
         }),
+        generateInName: Flags.boolean({
+            description: "Includes .generated in the generated type test filenames.",
+            default: true,
```

Comment:
From comment i understood default is false.

Reply:
I clarified this in the PR description (emphasis added):

> The default for the old CLI is false, so files will continue to have their old names by default. **The default for the new CLI is true, but it can be set to false manually when needed while we transition to the new CLI. **